### PR TITLE
Update appdata.xml

### PR DESCRIPTION
**Pull request by corecontingency**
Hasn't been updated since 0.9.8, so here you go!

### CODE REVIEW

**Comment by tehhowch** [comment]
Thanks!
Are you interested in contributing a script to convert our updates to the changelog into the appropriate appdata.xml updates? (It would go in the currently nonexistent `utils` folder.)

File: endless-sky.appdata.xml [hunk 1]
```diff
@@ -42,6 +42,542 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="0.9.11" date="2020-02-24">
```
Comment (tehhowch) [inline endless-sky.appdata.xml]: The odd releases should include `type="development"` and perhaps also `urgency="low"` per the spec.

**Comment by corecontingency** [comment]
> Are you interested in contributing a script to convert our updates to the changelog into the appropriate appdata.xml updates? (It would go in the currently nonexistent utils folder.)

I am not sure how much I would be able to help, I don't know any programming languages (yet), I just updated the appdata.xml manually (with some assistance from vim, admittedly).

**Comment by tehhowch** [comment]
I switched this to an abbreviated summary, which is a lot more feasible, and added in url links to the full changelog.

**Pull request merged**